application/controllers/admin/tokens.php

```diff
@@ -902,6 +908,13 @@ function delete($iSurveyID)
             self::_newtokentable($iSurveyID);
         }
 
+        $token = Token::model($iSurveyID)->find('tid=' . $sTokenIDs);
```

Comment:
This breaks when `$sTokenIDs` is a list of ids (mark more than one token in token list and click delete). Please fix.